agentMET4FOF/metrological_streams.py

```diff
@@ -256,3 +257,75 @@ def _sine_wave_function(self, time, sine_freq):
         amplitude = np.sin(np.multiply(2 * np.pi * sine_freq, time))
         amplitude += np.random.normal(0, self.value_unc, amplitude.shape)
         return amplitude
+
+
+class MetrologicalMultiWaveGenerator(MetrologicalDataStreamMET4FOF):
+    """
+    Class to generate data as a sum of cosine wave and additional Gaussian noise.
+    Values with associated uncertainty are returned.
+
+    Parameters
+    ----------
+    sfreq:     float
+                sampling frequency which determines the time step when next_sample is called.
+    intercept: float
+                constant intercept of the signal
+    freq_arr:  np.ndarray of float
+              array with frequencies of components included in the signal
+    ampl_arr:  np.ndarray of float
+              array with amplitudes of components included in the signal
+    phase_ini_arr:  np.ndarray of float
+              array with initial phases of components included in the signal
```

Comment:
This should be formatted according to the [Numpy docstring style guide](https://numpydoc.readthedocs.io/en/latest/format.html#sections) which already takes into consideration the [PEP8 documentation string section](https://www.python.org/dev/peps/pep-0008/#documentation-strings).

Notice the spacing of type specifications and the first lines.

```suggestion
    """Class to generate data as a sum of cosine wave and additional Gaussian noise.
    
    Values with associated uncertainty are returned.

    Parameters
    ----------
    sfreq : float
        sampling frequency which determines the time step when next_sample is called.
    intercept : float
        constant intercept of the signal
    freq_arr : np.ndarray of float
        array with frequencies of components included in the signal
    ampl_arr : np.ndarray of float
        array with amplitudes of components included in the signal
    phase_ini_arr : np.ndarray of float
        array with initial phases of components included in the signal
```